api/src/main/scala/dcos/metronome/api/ErrorHandler.scala

```diff
@@ -34,3 +35,7 @@ class ErrorHandler extends HttpErrorHandler {
     */
   private def escape(msg: String): String = HtmlFormat.escape(msg).body
 }
+
+object ErrorHandler {
+  val noRouteHandlerMessage = "Unsupported route, no handler found."
```

Comment:
lets change the message here...   perhaps "Unknown route." is sufficient

reasoning:
1. it seems direct and to the point and is what a 404 is.   
2. I don't like exposing internal details to an http client...  "no handler found" seems like an internal concern.

Reply:
otherwise looks good

Reply:
"Unknown route." sounds sufficient to me as well